apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/BindingTests.cs

```diff
@@ -40,65 +41,71 @@ public class BindingTests
             SpannerDbType.Numeric,
             SpannerDbType.Date,
             SpannerDbType.Bytes,
-            SpannerDbType.Json,
             SpannerDbType.ArrayOf(SpannerDbType.Bool),
             SpannerDbType.ArrayOf(SpannerDbType.String),
             SpannerDbType.ArrayOf(SpannerDbType.Int64),
             SpannerDbType.ArrayOf(SpannerDbType.Timestamp),
             SpannerDbType.ArrayOf(SpannerDbType.Float64),
             SpannerDbType.ArrayOf(SpannerDbType.Numeric),
             SpannerDbType.ArrayOf(SpannerDbType.Date),
-            SpannerDbType.ArrayOf(SpannerDbType.Bytes),
+            SpannerDbType.ArrayOf(SpannerDbType.Bytes)
+        };
+
+        // These SpannerDbTypes are unsupported on emulator.
+        public static TheoryData<SpannerDbType> BindUnsupportedNullData { get; } = new TheoryData<SpannerDbType>
+        {
+            SpannerDbType.Json,
             SpannerDbType.ArrayOf(SpannerDbType.Json),
         };
 
+        // TODO: xUnit v3 supports traits for DataAttributes. Use that instead of Skip when we migrate. 
         [SkippableTheory]
         [MemberData(nameof(BindNullData))]
+        [MemberData(nameof(BindUnsupportedNullData))]
         public async Task BindNull(SpannerDbType parameterType)
         {
             Skip.If(_fixture.RunningOnEmulator && (SpannerDbType.Json.Equals(parameterType) || SpannerDbType.ArrayOf(SpannerDbType.Json).Equals(parameterType)), "The emulator does not support the JSON type");
-            using (var connection = _fixture.GetConnection())
-            {
-                var cmd = connection.CreateSelectCommand("SELECT @v");
-                cmd.Parameters.Add("v", parameterType, null);
-                using (var reader = await cmd.ExecuteReaderAsync())
-                {
-                    Assert.True(await reader.ReadAsync());
+            var cmd = GetSelectCommand(parameterType, null);
+            await GetReaderAndAssertNull(cmd);
+        }
 
-                    Assert.True(reader.IsDBNull(0));
-                    Assert.Equal(DBNull.Value, reader.GetValue(0));
+        internal static async Task GetReaderAndAssertNull(SpannerCommand command)
+        {
+            using var reader = await command.ExecuteReaderAsync();
```

Comment:
I think it's better if this is done by the callers and this method just asserts the null.